SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -476,17 +476,13 @@ bool USpatialActorChannel::ReplicateSubobject(UObject* Object, FClassInfo& Info,
 bool USpatialActorChannel::ReplicateSubobject(UObject* Obj, FOutBunch& Bunch, const FReplicationFlags& RepFlags)
 {
 	// Intentionally don't call Super::ReplicateSubobject() but rather call our custom version instead.
-	if (FClassInfo* SubobjectInfo = NetDriver->TypebindingManager->FindClassInfoByObject(Obj))
-	{
-		return ReplicateSubobject(Obj, *SubobjectInfo, RepFlags);
-	}
-
-	return false;
+	const FClassInfo& SubobjectInfo = NetDriver->ClassInfoManager->GetClassInfoByObject(Obj);
```

Comment:
`FindClassInfoByObject` return null before was a valid case of some custom replication logic (e.g. AbilitySystem) calling `ReplicateSubobject` on unsupported objects. I suspect that may result in a crash now.